config_system/tests/formatter/expected.out

```diff
@@ -0,0 +1,69 @@
+# Some comment that need to be parsed!
+
+source "foo"
+
+config MODE
+	bool "Mode"
+	default n
+
+
+choice
+	prompt "Choose"
+	default OPTION_A
+
+config OPTION_A
+	bool "A"
+
+config OPTION_B
+	bool "B"
+	depends on MODE
+
+config OPTION_C
+	bool "C"
+	depends on MODE
+
+endchoice
+choice
+    int 365 if HOW_MUCH_DAYS_IN_YEAR || !WEEKS && DAYS
+    default BUILDER_BUILD if DROID
+	prompt "Choice 1" if ALA_MA_KOTA
```

Comment:
OK, so we're not standardizing indentation of lines yet. That's OK for this commit where I'd like to get the basic infrastructure in place. Do we think that we can do that with what we have (as this is going to be the most common problem)

Reply:
Yes changing spaces -> tabs or tabs -> spaces would require deeper changes in lex.py and it's for sure achievable but I guess it would require changes in YACC parser also, which will be harder than this change I guess

Reply:
I took a deeper look. Just for addressing this kind of indentation difference, I don't think we need to touch the lexer or grammar: we should be able to figure out what's happening in mconfigfmt.py and apply appropriate formatting directly there.